src/gateways/createBuyPrintConfigLoader.ts

```diff
@@ -1,9 +1,10 @@
 import { rejectResponse } from '../helpers/fetch';
+import Sentry from '../helpers/Sentry';
 
 export interface BuyPrintResponse {
   buy_urls: Array<{
     url: string;
-    provider: 'amazon';
+    provider: 'amazon' | 'openstax fallback';
```

Comment:
I'm not sure if this matters but I think we should add `_` between words to match API response style > `openstax_fallback`